x-pack/plugins/log_explorer/tsconfig.json

```diff
@@ -37,7 +37,9 @@
     "@kbn/unified-data-table",
     "@kbn/unified-field-list",
     "@kbn/unified-search-plugin",
-    "@kbn/xstate-utils"
+    "@kbn/xstate-utils",
+    "@kbn/ecs",
+    "@kbn/react-field"
```

Comment:
<img width="629" alt="Screenshot 2023-12-13 at 09 03 15" src="https://github.com/elastic/kibana/assets/1532934/beea0e62-db7a-42b6-b089-42cb6a106399">

Oh damn, those full field definitions from the `@kbn/ecs` package sure are something...

Reply:
Yeah, even tree-shaking the package (the diff before was around +900kb non-minified, down to 600 now) it still adds a lot of code on the async bundle.
However, this is a temporary solution, in following iterations we expect to consume an external service to retrieve the details of a field.

Reply:
we, @elastic/kibana-data-discovery , are also interested in consuming `kbn/ecs` package, I think we could consider to add it to `kbn-ui-shared-deps-src/BUILD.bazel`. I think it would be a good fit here, since it's also code that doesn't change too often (it's generated, right?)